sdk/keyvault/keyvault-certificates/test/recoverBackupRestore.test.ts

```diff
@@ -85,7 +85,16 @@ describe("Certificates client - restore certificates and recover backups", () =>
     );
     const backup = await client.backupCertificate(certificateName);
     await testClient.flushCertificate(certificateName);
-    await retry(async () => client.restoreCertificateBackup(backup!));
+    while (true) {
+      try {
+        await client.restoreCertificateBackup(backup as Uint8Array);
+        break;
+      } catch (e) {
+        console.log("Can't restore the certificate since it's not fully deleted:", e.message);
+        console.log("Retrying in one second...");
+        await delay(1000);
```

Comment:
should we give up after a max number of tries?

Reply:
It will happen automatically. We specify a timeout per test case on Mocha through the parameters sent on the package.json scripts.